orchestrator/careplancontributor/service.go

```diff
@@ -259,7 +244,43 @@ func (s Service) withSessionOrBearerToken(next func(response http.ResponseWriter
 	}
 }
 
-func (s Service) handleNotification(_ any) error {
-	log.Info().Msg("TODO: Handle received notification")
+func (s Service) handleNotification(resource any) error {
+	notification := resource.(*coolfhir.SubscriptionNotification)
+	if notification == nil {
```

Comment:
this will only be `true` if `notification` is actually `nil`, if it's something other, the cast will panic.

Reply:
Ah fair point! I will add some additional checks and tests